astropy/coordinates/tests/test_api_ape5.py

```diff
@@ -31,6 +31,8 @@
 else:
     HAS_SCIPY = True
 
+raises = pytest.raises
```

Comment:
In every other file we seem to import `raises` from `tests.helper` - maybe do that here too? Alternatively, I'd still prefer `from pytest import raises`.

Reply:
Indeed, I'll change this to be consistent with the rest, to import from tests.helper.